knowledge-base/importing-fiddler-classic-settings.md

```diff
@@ -0,0 +1,68 @@
+
+The comparison table showcases several reasons why you would like to migrate from Fiddler Classic to the new Fiddler Everywhere. In this article, we will provide information about the built-in import wizard available in Fiddler Everywhere, which enables you to quickly and efficiently import your settings, rules, and filters from the old Fiddler Classic.
+
+## Importing Application Options
+
+Fiddler Everywhere version 5.2.0 provides a built-in wizard allowing you to import the following configuration options from Fiddler Classic automatically. All configuration options have full feature parity meaning they behave exactly the same in both applications. Once imported the configuration options can be found in the **Settings** menu in Fiddler Everywhere.
+
+The import wizard is accessible through the **Home** pane in Fiddler Everywhere.
+![import configuration options](../images/import-from-classic/import-options.png)
+
+- Fiddler Classic certificate&mdash;Instructs Fiddler Everwhere to use the installed Fiddler Classic certificate authority file. This option is available only if you have not yet installed and trusted the Fiddler Everywhere CA.
+
+- Decrypt HTTPS traffic&mdash;Exports the Fiddler Classic value from **Tools > Options > HTTPS > Decrypt HTTPS Traffic** and imports as value for **Settings > HTTPS > Capture HTTPS Traffic** option in Fiddler Everywhere.
+
+- Ignore server certificate errors&mdash;Exports the Fiddler Classic value from **Tools > Options > HTTPS > Ignore server certificate errors (unsafe)** and imports as value for **Settings > HTTPS > Ignore server certificate errors (unsafe)** option in Fiddler Everywhere.
+
+- Listening proxy port&mdash;Exports the Fiddler Classic value from **Tools > Options > Connection > Fiddler Classic listens on port** and imports as value for **Settings > Connections > Fiddler listens on port** option in Fiddler Everywhere.
+
+- Bypassed URLs&mdash;Exports the Fiddler Classic value from **Tools > Options > Connections > Bypass Fiddler Classic for URLs that starts with** and imports as value for **Settings > Connections > Bypass Fiddler for URLs that starts with** option in Fiddler Everywhere.
+
+- Gateway settings&mdash;Exports the Fiddler Classic value from **Tools > Options > Gateway** and imports as value for **Settings > Gateway** option in Fiddler Everywhere.
+
+- Allow remote computers to connect&mdash;Exports the Fiddler Classic value from **Tools > Options > Connections > Allow Remote Computers to connect** and imports as value for **Settings > Connections > Allow remote devices to connnet** option in Fiddler Everywhere.
+
+## Importing AutoResponder Rules
+
+In Fiddler Everywhere, the Classic's **AutoResponder** (as depicted in the screenshot below) is referred to as **Rules**. You can import only the loaded AutoResponder rules from Fiddler Classic to Fiddler Everywhere. Once imported, you'll be able to modify or delete these rules as needed.
+
```

Comment:
Here we can add a note that any saved rules in .farx archives can also be imported at any point in time through the "Import" button in Fiddler Everywhere's Rules tab.

Another thing we can mention is that if the rules in Fiddler Classic have a "Comment", it will be used for the rules name in Fiddler Everywhere. Any Fiddler Classic rules that have no comment will use a predefined name.

Reply:
Should we show a screenshot with where the user can find the imported rules in Fiddler Everywhere, and how they will look?